BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -53,8 +53,8 @@
       <EnvironmentVariables>
          <EnvironmentVariable
             key = "BENCHMARK_RUN"
-            value = "instrumented"
-            isEnabled = "YES">
+            value = "metrics"
+            isEnabled = "NO">
```

Comment:
Will there still be instrumented scenario?

Reply:
This change was not intented, I reverted 👍

Reply:
My question remains though: does this change from `instrumented` set to `YES` to `NO` mean there will be no instrumented benchmark anymore?

Reply:
No, this is the env variable for running this app in Debug:
<img width="923" alt="image" src="https://github.com/user-attachments/assets/efc6485d-8124-4e62-9b91-c586c37627c8">

When set to `NO` (unselected), the `none` run will be selected. This is for development only, to simply select a run and a scenario, it is not applied to s8s build.

Reply:
Got it, thanks!